decisions/Gondor.txt

```diff
@@ -1845,6 +1868,77 @@ political_decisions = {
 		}
 	}
 	
+	claim_umbar_GON = {
+		picture = "umbar_map"
+		potential = {
+			tag = GON
+			OR = {
+				AND = {
+					UMB_256 = { owned_by = GON }
+					UMB_260 = { owned_by = GON }
+					UMB_263 = { owned_by = GON }
+					UMB_270 = { owned_by = GON }
+					UMB_273 = { owned_by = GON }
+					HAR_277 = { owned_by = GON }
+				}
+				AND = {
+					OR = {
+						UMB_256 = { owned_by = GON }
+						UMB_260 = { owned_by = GON }
+						UMB_263 = { owned_by = GON }
+						UMB_270 = { owned_by = GON }
+						UMB_273 = { owned_by = GON }
+						HAR_277 = { owned_by = GON }
+					}
+					UMB = { exists = no }
+				}
+			}
+			NOT = {
+				has_country_modifier = low_military_recruitment
+				is_core = 256
+				is_core = 257
+				is_core = 258
+				is_core = 259
+				is_core = 260
+				is_core = 282
+				is_core = 264
+				is_core = 261
+				is_core = 262
+				is_core = 265
+				is_core = 266
+				is_core = 263
+				is_core = 270
+				is_core = 267
+				is_core = 268
+				is_core = 273
+				is_core = 271
+				is_core = 272
+				is_core = 274
+				is_core = 275
+				is_core = 276
+				is_core = 278
+				is_core = 280
+				is_core = 283
+				is_core = 318
+				is_core = 319
+			}
```

Comment:
Update the `NOT = { is_core }` to one of these checks per region:

```
# region = { conditions } returns true if any province in the region has the condition. NOT = { condition } checks the oposite of the condition, and NOT { region = { NOT = { condition } } } checks the oposite of any provinces not having the condition. This is logically equivalent to all provinces having the condition
# In standard notation (will work extrapolated out for one variable = one province in region): ¬(¬A ∪ ¬B ∪ ¬C) = ¬¬A ∩ ¬¬B ∩ ¬¬C = A ∩ B ∩ C
NOT = { region = { NOT = { is_core = THIS } } }
```

Reply:
Thanks.

Reply:
Wait, shouldn't it be NOT = { region = { is_core = THIS } }

Reply:
That'll return true if GON doesn't have cores in any province of the region -- if they have a core in one province in the region, `NOT = { region = { is_core = THIS } }` will return false. I imagine that they should be getting cores on any provinces in these regions that they don't already have
¬(A ∪ B ∪ C) = ¬A ∩ ¬B ∩ ¬C

Reply:
Made a commit